packages/docusaurus-theme-classic/src/theme/Navbar/index.tsx

```diff
@@ -121,10 +117,7 @@ function Navbar(): JSX.Element {
               onChange={onToggleChange}
             />
           )}
-          <SearchBar
-            handleSearchBarToggle={setIsSearchBarExpanded}
-            isSearchBarExpanded={isSearchBarExpanded}
```

Comment:
It's dead code, although it may be used by third party plugins implementing local search (not Algolia). It doesn't make sense to support it anyway.